### PR TITLE
Embed PEM of root certificates in Mozilla's root store

### PR DESCRIPTION
**Pull request by saimanoj01**
### What this PR does / why we need it
We are [embedding in the core CLI](https://github.com/vmware-tanzu/tanzu-framework/blob/main/pkg/v1/cli/clientconfighelpers/windows_trusted_ca.go) the root CA cert for [projects.registry.vmware.com](http://projects.registry.vmware.com/) - however the cert is the Root CA for the internal VIP and not the public access via CDN. [Tanzu CLI writes the embedded root CA cert file to the home directory on Windows and then adds it to the cert chain](https://github.com/vmware-tanzu/tanzu-framework/blob/fe23cf4d8a1d33251ebf2599b408419122a34808/pkg/v1/cli/clientconfighelpers/helpers.go#L67), which is why no-one using the CLI on Windows other than internal to VMware can download any plugins or do a tanzu init.

The fix is to include the full mozilla bundle - [PEM of Root Certificates in Mozilla's Root Store with the Websites (TLS/SSL) Trust Bit Enabled](https://ccadb-public.secure.force.com/mozilla/IncludedRootsPEMTxt?TrustBitsInclude=Websites)

### Which issue(s) this PR fixes

Fixes https://github.com/vmware-tanzu/tanzu-framework/issues/1710

### Describe testing done for PR
1. Made sure that the `tanzu plugin sync` works from a windows machine setup on Azure.
2. Made sure that `tanzu management-cluster create` downloads the bom files necessary to proceed with the cluster creation

### Release note

```release-note

```

### PR Checklist

- [ ] Squash the commits into one or a small number of logical commits
      
- [ ] Use good commit [messages](https://github.com/vmware-tanzu/tanzu-framework/blob/main/CONTRIBUTING.md)
- [ ] Ensure PR contains terms all contributors can understand and links all contributors can access

### CODE REVIEW

**Review by tenczar — APPROVED** [review]
LGTM. I'm a little annoyed that the Mozilla wiki does not seem to have a hash for downloading this file. Getting a rogue CA slipped into a giant list of trusted root servers seems like something that should be guarded against. In any case the change is straight forward.

File: pkg/v1/cli/clientconfighelpers/windows_trusted_ca.go [hunk 1]
```diff
@@ -10,31 +10,3546 @@ package clientconfighelpers
 //	-showcerts
```
Comment (prkalle) [inline pkg/v1/cli/clientconfighelpers/windows_trusted_ca.go:10]: nit: comment to be updated and please mention the url from where we downloaded the mozilla cert bundle and a note to update this bundle periodically would be helpful.

**Review by prkalle — APPROVED** [review]
LGTM. Thanks for the changes @saimanoj01 .
I have one minor nit comment.

**Review by anujc25 — APPROVED** [review]
LGTM

**Pull request merged**